scripts/ci/setup_hadoop_env.sh

```diff
@@ -42,7 +42,7 @@ while test $# -gt 0; do
     esac
 done
 
-HADOOP_HOME=/tmp/hadoop-${HADOOP_DISTRO}
+HADOOP_HOME=${TOX_CACHE_FOLDER}/hadoop_cache/hadoop-${HADOOP_DISTRO}
```

Comment:
You can also add this folder to Travis as a cache dir, but I'm not sure if there's too much to gain.